recipes/waf/all/test_package/conanfile.py

```diff
@@ -9,11 +9,16 @@ class TestPackageConan(ConanFile):
     exports_sources = "a.cpp", "b.cpp", "main.c", "main.cpp", "wscript"
 
```

Comment:
Perhaps this should contain `test_type = "build_requires"` + `required_conan_version = ">=1.36.0"`?

https://docs.conan.io/en/latest/devtools/build_requires.html#testing-build-requires

Reply:
it may contain that, still, `waf.exe` will be ARMv8 and it will fail?

Reply:
`waf` is python-only. There is no binary that I'm aware of.
This recipe still uses `os_build` and `arch_build`.
Perhaps switching to `os`/`arch` will improve cross builds?

Reply:
it should, I am not a fan of `os_build`/`arch_build`